azure-pipelines.yml

```diff
@@ -126,21 +126,21 @@ jobs:
       frameworks: # for private jobs we want to benchmark .NET Core 3.0 only
         - netcoreapp3.0
         
-# # Ubuntu 1604 ARM64 micro benchmarks, public correctness job
-# - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-#   - template: /eng/performance/benchmark_jobs.yml
-#     parameters:
-#       osName: ubuntu
-#       osVersion: 1604
-#       kind: micro
-#       architecture: arm64
-#       pool: Hosted Ubuntu 1604
-#       queue: Ubuntu.1604.Arm64.Open
-#       container: ubuntu_1604_arm64_cross_container
-#       csproj: src/benchmarks/micro/MicroBenchmarks.csproj
-#       categories: ['coreclr', 'corefx']
-#       frameworks: # currently ARM64 is supported only by .NET Core 3.0 https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md
-#         - netcoreapp3.0
+# Ubuntu 1604 ARM64 micro benchmarks, public correctness job
+- ${{ if eq(variables['System.TeamProject'], 'public') }}:
+  - template: /eng/performance/benchmark_jobs.yml
+    parameters:
+      osName: ubuntu
+      osVersion: 1604
+      kind: micro
+      architecture: arm64
+      pool: Hosted Ubuntu 1604
+      queue: Ubuntu.1604.Arm64.Open
+      container: ubuntu_1604_arm64_cross_container
+      csproj: src/benchmarks/micro/MicroBenchmarks.csproj
+      categories: ['coreclr', 'corefx']
```

Comment:
Don't we want to run `mldotnet` on ARM64 too?

Reply:
ML.NET has a lot of native dependencies, I don't know if ARM is supported

Reply:
We should find out, but this can be done separately.

Reply:
It's not supported yet https://github.com/dotnet/machinelearning/issues/1662#issuecomment-440378848